draft-deepspace-ip-architecture.xml

```diff
@@ -216,6 +216,23 @@
+          <li><t>Connection Initiation / Handshaking - Due to the need to be robust to stale packets, errors, and denial of service attacks, Internet transports include handshaking state machines, such as 3-way and 4-way handshakes for connection establishment (the case can be even worse for some transport protocol and TLS combinations).  Since the interplanetary round trip times may be larger than the duration of contact periods, these handshaking mechanisms are very inefficient.  Though they are tolerable in cases such as between Earth and lunar networks, they are stifling for Earth-Mars and other interplanetary network paths.  Transports, such as QUIC, that have the ability to resume based on shared state from prior application connections or to rapidly start transferring data ("0RTT") can be suitably efficient, once initial state is obtained; however, they still require a complete initial handshake with the full set of round trip times imposed.  For interplanetary use, it may be beneficial to find ways to securely pre-set information to allow this more efficient startup, without requiring the full initial handshake even once.</t></li>
+          <li><t>Capability / Feature Negotiation - Any negotiation of additional features that requires full round trip times is prohibitive for general interplanetary use, espcially if data transmission is stalled while negotiation takes place.  Commonly, much negotiation can occur in the initial connection handshake.  It will be beneficial if transport stacks can cache (or securely pre-configure caches) of pre-negotiated features with typical hosts that they plan to communicate with during the course of a mission.</t></li>
```

Comment:
Similarly, pre-launch negotiation would be helpful.

Reply:
+1, though not 100% static / set-in-stone -- must be manageable